File: JS/Advanced/Classes/LAB/Person.js
function solve() {
    class Person {
        constructor(name, lastName, age, email) {
            this.firstName = name;
            this.lastName = lastName;
            this.age = age;
            this.email = email;
        };

        toString() {
            return `${this.firstName} ${this.lastName} (age: ${this.age}, email: ${this.email})`;
        }
    };

    let ana = new Person('Anna', 'Simpson', 22, 'anna@yahoo.com');
    let softUni = new Person('SoftUni');
    let stephan = new Person('Stephan', 'Johnson', 25);
    let gabriel = new Person('Gabriel', 'Peterson', 24, 'g.p@gmail.com');

    return [ana, softUni, stephan, gabriel];
};

File: OOP/Encapsulation/LAB/PersonalInfo/Person.cs
namespace PersonsInfo
{
    public class Person
    {
        private string name;
        private string lastName;
        private int age;

        public Person(string name, string lastName, int age)
        {
            this.Name = name;
            this.LastName = lastName;
            this.Age = age;
        }

        public string Name { get => this.name; private set => this.name = value; }

        public string LastName { get => this.lastName; private set => this.lastName = value; }

        public int Age { get => this.age; private set => this.age = value; }

        public override string ToString()
        {
            string output = $"{this.Name} {this.LastName} is {this.Age} years old.";
            return output;
        }
    }
}
